.gitignore

```diff
@@ -3,3 +3,6 @@
 pipeline-bundle-list
 task-bundle-list
 bundle_values.env
+
+*~
```

Comment:
Out of curiosity, what are these?

Reply:
These are temporary and backup files created by some text editors like vim.